Algebra.java

```diff
@@ -21,47 +22,193 @@ public static void main(String args[]) {
    		System.out.println(sqrt(36));
 		System.out.println(sqrt(263169));
    		System.out.println(sqrt(76123));
-	}  
 
+		
+	}  
+	public static int abs(int x){
+		if (x<0) {
+			return -x; 	
+		}
+		return x;
+	}
+	public static int minusabs(int x){
+		if(x>0){
+			return -x;
+		}
+		return x;
+	}
 	// Returns x1 + x2
 	public static int plus(int x1, int x2) {
-		// Replace the following statement with your code
-		return 0;
+	if (x1>=0&&x2>=0) {
+		for(int i = 0 ; i<x1; i++ ){
+			x2++;
+		}
+		return x2;
 	}
+	else{
```

Comment:
A lot of "sub-if"s...
This could be solved by a much shorter, cleaner approach.
I recommend you to try and think of a more elegant solution!